hack/source-manifests.sh

```diff
@@ -33,8 +33,8 @@ mkdir -p $OUTDIR_TOOLS
 
 # ==== DUMP NOOBAA YAMLS ====
 function dump_noobaa_csv() {
-	noobaa_dump_crds_cmd="crd yaml"
-	noobaa_dump_csv_cmd="olm csv yaml"
+	noobaa_dump_crds_cmd="/usr/local/bin/noobaa-operator crd yaml"
```

Comment:
Interesting. Has anything in the noobaa image changed that requires this change?
Just seeing this line change, I am think that this can not have worked before...

Reply:
I took another look, and I will update this PR to use newer images.

we had [this PR](https://github.com/noobaa/noobaa-operator/pull/556) that removed the entrypoint. this broke the dump command. 
since then it was fixed [here](https://github.com/noobaa/noobaa-operator/pull/575), but `noobaa-operator:master-20210308` does not contain the fix. 
we pushed `noobaa/noobaa-operator:master-20210419` yesterday, and it does contain the second PR, so I'll use that one instead. 
I think that I'll add `--entrypoint=/usr/local/bin/noobaa-operator` to the command, just to avoid this in the future